src/runtime/validator.ts

```diff
@@ -128,6 +156,9 @@ function validateAndParseValueAgainstSchema (
   }
   // Strings
   if (currentSchema.type === 'string') {
+    if (value instanceof Date && currentSchema.format === 'date-time') {
+      value = value.toISOString()
```

Comment:
IMO we should expect a Date object when format=date-time

Reply:
JSON.stringify implicitly converts Date to string, but not the other way around.

So, any Date object RECEIVED as parameter will always be a string. 

Moreover, the validation needs to turn Date into a string because Date objects are not valid openAPI response types...

Reply:
okay, there is another aspect to this: I think currently (not related to my PR), inbound Date-types are broken since they are delivered as "strings".

Should we automatically convert any "date-time" string into a Date object during validation?

Reply:
I was wrong: Inbound date-time strings were converted to Date.

I change the code now to pass-through `value` if its of type `Date` and a date-time string is expected.